### PR TITLE
[WIP] *: switch to go mod

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@runcom, I prefer to wait until modules are considered stable. It looks like this will happen with the release of go v1.12 which will be released at the end of January + waiting until it's packaged etc.

**Comment by runcom** [comment]
yes, we're not merging this right now, but we need it tested and report upstream any bug we find.

File: README.md [hunk 1]
```diff
@@ -231,6 +231,8 @@ CONTRIBUTING
 
 ### Dependencies management
 
+### TODO(runcom): switch to go mod
+
 Make sure [`vndr`](https://github.com/LK4D4/vndr) is installed.
```
Comment (giuseppe) [inline README.md]: will we still need `vndr`?

Comment (runcom) [reply README.md]: nope, there's a todo above this :)

**Comment by rhatdan** [comment]
Adding WIP flag to make sure no one accidentally merges.

**Comment by rhatdan** [comment]
@vrothberg Can you take this one over.

**Comment by vrothberg** [comment]
> @vrothberg Can you take this one over.

Sure. I will take the crusade also over to libpod and buildah and eventually down to image and storage. But I need some free cycles for that. I am closing this PR in favor of the future one.

**Pull request closed without merging**